### PR TITLE
PP-682 tracking master of cardid data

### PR DESCRIPTION
**Pull request by imranbohoran**
## WHAT
_A brief description of the pull request:_
Tracking master changes for cardid-data

## WHO
_Who should review this pull request:_

- [x] Developers
- [] WebOps

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -1,3 +1,4 @@
 [submodule "data"]
 	path = data
 	url = git@github.com:alphagov/pay-cardid-data.git
+    branch = master
```
Comment (PauloPortugal) [inline .gitmodules]: https://git-scm.com/docs/gitmodules
According to the documentation is always defaults to `master`

Comment (imranbohoran) [inline .gitmodules]: Advice on submodules on the internet is to add the branch so it can track it and not get detached heads. Hence this change.

**Comment by PauloPortugal** [comment]
rebuild

**Comment by imranbohoran** [comment]
rebuild

**Comment by PauloPortugal** [comment]
:+1:

**Pull request merged**